### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by Joozty**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 CORE
```
Comment (viktor-yakubiv) [inline LICENSE:3]:
```suggestion
Copyright (c) 2020 CORE Team <theteam@core.ac.uk>
```

I am not sure about this, we should consult with whole team @oacore/services. I used 'Knowledge Media Institute, The Open University' before but I am not sure about this neither.

**Comment by viktor-yakubiv** [comment]
I am not sure we will get any feedback. Let's merge this using dev@core.ac.uk email instead of the one you listed.

**Comment by Joozty** [comment]
@petrknoth asked us to wait so let's wait.

**Comment by viktor-yakubiv** [comment]
Sorry, sure, this is not urgent. @petrknoth do you have any better suggestion for the license?

**Pull request closed without merging**